<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyObjectsClass</class>
 <widget class="QMainWindow" name="EasyObjectsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyObjects</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::InputMethodHint::ImhNone</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ObjectNameRegex">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>single: 'prop_stunt_*'</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ObjectNameRegexLabel">
       <property name="text">
        <string>Object Name Regex:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ObjectNamesIgnoreLabel">
       <property name="text">
        <string>Object Names Ingore:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ObjectNamesIgnore">
       <property name="toolTip">
        <string extracomment="wow"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>multi: 'prop1, prop2'</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ObjectDecalsIgnoreLabel">
       <property name="text">
        <string>Object Decals Ignore:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ObjectDecalsIgnore">
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>multi: '6, 12, 2'</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QComboBox" name="ObjectCopyMode">
       <item>
        <property name="text">
         <string>Mirror Quadrant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mirror Axes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spin Rotate</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Input File:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Save Output:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="SaveYmap">
       <property name="text">
        <string>YMAP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="SaveJson">
       <property name="text">
        <string>JSON</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QPushButton" name="SelectFile">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>9</y>
      <width>561</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCustomPlot" name="ObjectPlot" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="SourceQuadrant">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>single: '2'</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="SourceQuadrantLabel">
         <property name="text">
          <string>Source Quadrant:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="DestinationQuadrantsLabel">
         <property name="text">
          <string>Destination Quadrants:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="DestinationQuadrants">
         <property name="toolTip">
          <string extracomment="wow"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>multi: '1, 3, 4'</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="RotationalOffsetLabel">
         <property name="text">
          <string>Rotational Offset (째):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="RotationalOffset">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>single: '-45.0'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="MirrorLineAngle">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>single: '0.0'</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="MirrorLineAngleLabel">
         <property name="text">
          <string>Mirror Line Angle (째):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="SourceSegmentAngleLabel">
         <property name="text">
          <string>Source Segment Angle (째):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="SourceSegmentAngle">
         <property name="toolTip">
          <string extracomment="wow"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>single: '45.0'</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="SourceSegmentSizeLabel">
         <property name="text">
          <string>Source Segment Size (째):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="SourceSegmentSize">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>single: '90.0'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="formLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Source Segment Angle (째):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="DestinationQuadrantsLabel_3">
         <property name="text">
          <string>Source Segment Size (째):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="DestinationQuadrants_3">
         <property name="toolTip">
          <string extracomment="wow"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>single: '90.0'</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="RotationalOffsetLabel_3">
         <property name="text">
          <string>Rotational Offset (째):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="RotationalOffset_3">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>single: '-45.0'</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="DestinationQuadrants_4">
         <property name="toolTip">
          <string extracomment="wow"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>single: '45.0'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="ObjectDecalSwapsLabel">
       <property name="text">
        <string>Object Decal Swaps:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ObjectDecalSwaps">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>multi: '1-6, 8-9, 12-4'</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuThis_is_like_a_uhhhh_app_or_something_ye">
    <property name="title">
     <string>This is like a uhhhh app or something ye</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuand_this_is_some_more_text_as_well">
    <property name="title">
     <string>and this is some more text as well</string>
    </property>
   </widget>
   <addaction name="menuThis_is_like_a_uhhhh_app_or_something_ye"/>
   <addaction name="menuand_this_is_some_more_text_as_well"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EasyObjects.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ObjectCopyMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_SelectFile_clicked()</slot>
 </slots>
</ui>
